includes/elements/container.php

```diff
@@ -443,6 +443,28 @@ protected function register_container_layout_controls() {
 			]
 		);
 
+		$this->add_responsive_control(
+			'max_height',
+			[
+				'label' => esc_html__( 'Max Height', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'vh', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 1440,
```

Comment:
Why limiting the `max-height` to `1440`? What if I need a larger container?

Reply:
Shall I use a high value or is there an unlimited option?

Reply:
I've raised the value to 10000. Otherwise the slide is very hard to control. User can always enter a higher value if needed.